src/doomtype.h

```diff
@@ -74,11 +74,25 @@ typedef byte lighttable_t;
 
 #define arrlen(array) (sizeof(array) / sizeof(*array))
 
-#define MIN(a, b) (((a) < (b)) ? (a) : (b))
-
-#define MAX(a, b) (((a) > (b)) ? (a) : (b))
+#if defined(HAVE_TYPEOF)
+ #define MIN(a,b)             \
+ ({                           \
```

Comment:
MSVC doesn't support `({ })`: https://gcc.gnu.org/onlinedocs/gcc/Statement-Exprs.html
So this macro only works with GCC/Clang. The previous version was correct, sorry.